client/cypress/e2e/entry-spec.cy.js

```diff
@@ -23,11 +26,25 @@ describe('template spec', () => {
     cy.get('.journal-text').should('be.visible').contains('Today I am feeling anxious because I have a big test coming up!')
   })
 
+  it('Should allow the user to edit their entry', () => {
+    cy.get('button.uni-btn').click()
+    cy.get('.journal-text-edit').should('be.visible').type(' But after I went on a walk I felt much more calm!')
+    cy.get(':nth-child(4) > :nth-child(2)').click()
+    cy.get('.journal-content').should('be.visible').contains('Today I am feeling anxious because I have a big test coming up! But after I went on a walk I felt much more calm!')
+    cy.get('a.uni-btn').click()
+  })
+
+  it('Should allow the use to cancel a journal entry edit and return the entry entry to its previous value', () => {
+    cy.get('button.uni-btn').click()
+    cy.get('.journal-text-edit').should('be.visible').type(' But after I went on a walk I felt much more calm!')
+    cy.get(':nth-child(4) > :nth-child(1)').click()
+    cy.get('.journal-content').should('be.visible').contains('Today I am feeling anxious because I have a big test coming up')
+  })
+
```

Comment:
I just love how thorough your testing is. You truly thought of everything. Thank you!!

Reply:
agreed!